lib/game.rb

```diff
@@ -0,0 +1,86 @@
+require "yaml"
+require_relative "validate"
+require_relative "show_content"
+require_relative "db_methods"
+require_relative "user"
+module NewSuperCodebreaker2021
+  class Game
+    include Validate
+    include ShowContent
+    include DBMethods
+
+    START_COMMANDS = %i[start rules stats exit].freeze
+    DIFFICULTY_COMMANDS = %i[easy medium hell exit].freeze
+    GUESS_COMMANDS = %i[hint rules exit].freeze
+    AFTER_GAME_COMMANDS = %i[start save exit].freeze
+    YES_NO_COMMANDS = %i[yes no].freeze
+
+    def chose_com(command)
```

Comment:
Maybe `choose_command` is a much better?

Reply:
done